x-pack/plugins/aiops/public/components/log_categorization/category_validation_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import {
+  FieldValidationResults,
+  CATEGORY_EXAMPLES_VALIDATION_STATUS,
+} from '@kbn/ml-category-validator';
+
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  validationResults: FieldValidationResults | null;
+}
+
+export const FieldValidationCallout: FC<Props> = ({ validationResults }) => {
+  if (validationResults === null) {
+    return null;
+  }
+
+  if (validationResults.overallValidStatus === CATEGORY_EXAMPLES_VALIDATION_STATUS.VALID) {
+    return null;
+  }
+
+  return (
+    <>
+      {validationResults !== null ? (
```

Comment:
Guess this ternary can be removed since the same check is done above and returns early.

Reply:
Updated in https://github.com/elastic/kibana/pull/162319/commits/914c43d19d45dbc61c567f791ec0b4562a8012cf